### ISSUE TITLE
https pasteboard.co cannot provide a secure connection

### ISSUE THREAD

**Issue by ChuckSavage** [JoelBesada/pasteboard#51]
Has the license for the secure connection expired?

I love this site, but today I cannot access it.

A separate issue, and I can post it as well. Any of your http (non-secure) links from GitHub to pasteboard.co are blocked by MalewareBytes as containing a trojan. I'm not willing to risk infection to see if that is an error.

The image links for old uploads still work.

Considering the code here on github is 4 years old, I'm guessing this site is no longer being maintained. Where else to post issues? You could update your README.md file to point people to the right location.
